src/toil/test/utils/utilsTest.py

```diff
@@ -68,8 +68,17 @@ def setUp(self):
             self.correctSort.sort()
 
     def tearDown(self):
+        cruft = ['failed.log', 'fileToSort.txt',
+                 'output.txt', 'sortedFile.txt',
+                 'succeeded.log']
+        if os.path.exists(self.tempDir):
+            shutil.rmtree(self.tempDir)
+        if os.path.exists('tmp/'):
```

Comment:
I like using rmtree rather than `rm -fr` above, but this line makes me a bit nervous. What if I (stupidly) had created my own folder in the toil directory called "tmp" where I (stupidly) kept something important? Or worse, what if I were running these tests from my home directory, or the root folder? A similar argument could go for the other special-cased cruft files, though that's stretching it.

I totally get the desire to clean up the cruft created by some of these tests, but surely it's better to clean up the cruft where it's created: making sure tempDir is always used for all test temp files. A stopgap may be to `chdir` to the self.tempDir directory in setUp. That might solve having to special case these stray files without having to change any other code.

Reply:
That's a good point.  I'll see if I can't improve it.  Thanks again.

Reply:
So the cruft generated was removed when the status PR code was removed. I've removed these steps and I'll create a new PR for status after this including this clean up step and ask for your review there if that's alright!

Reply:
Sounds great, thanks!